libraries/AP_BoardConfig/AP_BoardConfig_CAN.cpp

```diff
@@ -181,9 +181,11 @@ void AP_BoardConfig_CAN::init()
                 _drivers[i]._driver = _drivers[i]._pcan = new AP_PiccoloCAN;
 
                 if (_drivers[i]._driver == nullptr) {
-                    AP_BoardConfig::config_error("PiccoloCAN init failed");
+                    AP_HAL::panic("Failed to allocate PiccoloCAN %d\n\r", i + 1);
```

Comment:
why the change to a panic?

Reply:
Only to make it similar to the call a few lines up, for uavcan driver